lib/fetch/index.js

```diff
@@ -96,7 +93,7 @@ class Fetch extends EE {
 async function fetch (...args) {
   if (args.length < 1) {
     throw new TypeError(
-      `Failed to execute 'fetch' on 'Window': 1 argument required, but only ${args.length} present.`
+      `Failed to execute 'fetch': 1 argument required, but only ${args.length} present.`
```

Comment:
This is externally observable. I would prefer if the error is the exact same as in Chrome.